jack/pcm_jack.c

```diff
@@ -149,13 +164,16 @@ snd_pcm_jack_process_cb(jack_nframes_t nframes, snd_pcm_ioplug_t *io)
 	}
 
 	if (io->state == SND_PCM_STATE_RUNNING) {
-		areas = snd_pcm_ioplug_mmap_areas(io);
+		const snd_pcm_channel_area_t *areas = snd_pcm_ioplug_mmap_areas(io);
```

Comment:
this change can go into the previous commit.

Reply:
Done.